estate/models/estate_property.py

```diff
@@ -0,0 +1,55 @@
+from odoo import api, fields, models
+from datetime import timedelta
+
+today = fields.date.today()
+three_months_later = (today + timedelta(days=90)).strftime("%Y-%m-%d")
```

Comment:
Use relativeDelta instead for date time calculations and try to do it in the field itself than creating two separate variables here